src/components/SearchWithFilters.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <div class="position-relative">
+    <Search
+      :modelValue="modelValue"
+      @update:modelValue="input => $emit('update:modelValue', input)"
+      :placeholder="$t('searchPlaceholder')"
+    >
+      <template v-slot:searchWithFilters>
+        <UiDropdown
+          class="search-dropdown text-left"
+          @redirect="redirectSearch"
+          :items="searchOptions"
+        >
+          <span v-text="searchSelectedOption" />
+          <Icon name="arrow-down" class="ml-1" />
+        </UiDropdown>
+      </template>
+    </Search>
+  </div>
+</template>
+
+<script>
+export default {
+  props: ['modelValue'],
+  data() {
+    return {
+      searchOptions: [
+        {
+          text: this.$t('spaces'),
```

Comment:
You should pass translation key here and do the translation on the dropdown component, otherwise if you change language in the UI the text remain in the previous language.

Reply:
@bonustrack fixed it